docs/sources/guides/basic_concepts.md

```diff
@@ -4,117 +4,92 @@ description = "Grafana intro and concept guide"
+In most cases, Grafana is deployed with a single organization.
 
-Grafana supports a wide variety of internal and external ways for Users to authenticate themselves. These include from its own integrated database, from an external SQL server, or from an external LDAP server.
+Each organization can have one or more data sources.
 
-For more details please refer to [User Auth](/reference/http_api/#users)
+All dashboards are owned by a particular organization.
 
-### Row
+ > Note: Most metric databases do not provide per-user series authentication. This means that organization data sources and dashboards are available to all users in a particular organization.
 
-A Row is a logical divider within a Dashboard, and is used to group Panels together.
+Refer to [Organization roles]({{< relref "organization_roles.md" >}}) for more information.
 
-Rows are always 12 “units” wide. These units are automatically scaled dependent on the horizontal resolution of your browser. You can control the relative width of Panels within a row by setting their own width.
+## Panel
 
-We utilize a unit abstraction so that Grafana looks great on all screens both small and huge.
+The *panel* is the basic visualization building block in Grafana. Each panel has a Query Editor specific to the data source selected in the panel. The query editor allows you to extract the perfect visualization to display on the panel.
 
- > Note: With MaxDataPoint functionality, Grafana can show you the perfect amount of datapoints no matter your resolution or time-range.
+There are a wide variety of styling and formatting options for each panel. Panels can be dragged and dropped and rearranged on the Dashboard. They can also be resized.
 
-Utilize the [Repeating Rows functionality](/reference/templating/#repeating-rows) to dynamically create or remove entire Rows (that can be filled with Panels), based on the Template variables selected.
+Panels like the [Graph]({{< relref "graph.md" >}}) panel allow you to graph as many metrics and series as you want. Other panels like [Singlestat]({{< relref "singlestat.md" >}}) require a reduction of a single query into a single number.
 
-Rows can be collapsed by clicking on the Row Title. If you save a Dashboard with a Row collapsed, it will save in that state and will not preload those graphs until the row is expanded.
+Panels can be made more dynamic with [Dashboard Templating]({{< relref "templating.md" >}}) variable strings within the panel configuration. The template can include queries to your data source configured in the Query Editor.
 
-### Panel
+Panels can be [shared]({{< relref "share_panel.md" >}}) easily in a variety of ways.
 
-The Panel is the basic visualization building block in Grafana. Each Panel provides a Query Editor (dependent on the Data Source selected in the panel) that allows you to extract the perfect visualization to display on the Panel by utilizing the Query Editor
+## Query editor
```

Comment:
@torkelo , do we mean the Query Inspector?

Reply:
no, I mean the query builder/editor that each data source has (looks different for each). This UI component builds the query.